apps/src/lab2/views/Lab2ShareDialogWrapper.tsx

```diff
@@ -66,12 +69,24 @@ const Lab2ShareDialogWrapper: React.FunctionComponent<
   >(undefined);
 
   useEffect(() => {
-    if (channelId && projectType) {
-      getSubmissionStatus(channelId, projectType).then(response =>
-        setSubmissionStatus(response)
-      );
+    // Only signed-in users can submit projects to be considered for the featured project gallery.
+    if (channelId && projectType && isSignedIn) {
+      try {
```

Comment:
`try/catch` won't work with promise syntax; you'll need to either convert this to an `async` function or handled the rejected promise with a `.catch()` function